api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -301,4 +301,8 @@ private FhirConstants() {
 	public static final String REVERSE_INCLUDE_SEARCH_HANDLER = "_revinclude.search.handler";
 	
 	public static final String CONDITION_OBSERVATION_CONCEPT_UUID = "1284AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+
+	public static final String MAX_SEARCH_HANDLER = "max.search.handler";
+
+	public static final String LASTN_SEARCH_HANDLER = "lastn.search.handler";
```

Comment:
Should we make it as `LASTN_OBSERVATION_SEARCH_HANDLER` instead? We'll have a different implementation for lastn-encounters and thus would require different handlers for these two.